src/include/compat.h

```diff
@@ -14,8 +14,13 @@
 
 #include "acconfig.h"
 
+#define PROCPREFIX ""
```

Comment:
okay how about 
```c++
#if defined(__linux__)
#define PROCPREFIX
#endif

#if defined(__FreeBSD__)
#define PROCPREFIX "/compat/linux"
...
```

as the proc fs only exists on linux, and its counterpart linprocfs only on FreeBSD. i just  don't really like the unnecessarily defined empty string prefix for linux and `#undef`